src/Select/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import getClassName from 'getclassname';
+import Option from './option';
+
+export interface SelectProps {
+  /** 
+  * Array with elements to be displayed
+  */
+  options?: any,
+   /**
+   * onChange event handler triggered when an option is selected. 
+   * The index of the option selected within the options array is stored on event.target.dataset.index 
+   */
```

Comment:
please check indentation, it seems that you have a mix between 2 a 3 spaces.

Reply:
Indentation mix fixed https://github.com/bod-endava/e-toast-react/pull/6/commits/be0b0513a154144666b3592174a2e219ebeb9774